lib/internal/http2/core.js

```diff
@@ -74,6 +74,34 @@ const {
   HTTP_STATUS_SWITCHING_PROTOCOLS
 } = constants;
 
+const defaultSettings = new Uint32Array(binding.defaultSettingsArrayBuffer);
+const settingsBuffer = new Uint32Array(binding.settingsArrayBuffer);
+const sessionState = new Float64Array(binding.sessionStateArrayBuffer);
+const streamState = new Float64Array(binding.streamStateArrayBuffer);
```

Comment:
can you please add a comment here to describe how those are used? They are a bit unclear.

Also, I might worry a bit about those `Float64`, as it seems we are casting integers there.